packages/next/src/lib/metadata/resolvers/resolve-url.ts

```diff
@@ -17,10 +18,18 @@ function resolveUrl(
     return parsedUrl
   } catch (_) {}
 
-  if (!metadataBase)
+  if (!metadataBase) {
```

Comment:
I don't think this condition will happen given that we always provide a fallback?

Reply:
you can override metadataBase to `null`

Reply:
you could set it in your exported `metadata` in any page/layout which might override the default one, in this case you can error